config/application.rb

```diff
@@ -48,12 +48,6 @@ class Application < Rails::Application
     # Enable the asset pipeline
     config.assets.enabled = true
 
-    config.assets.precompile += %w[
```

Comment:
Moved to manifest.js

Reply:
Just some history on modernizing the JS test approach in frontend app:
- https://github.com/alphagov/frontend/pull/1414 - 2018
added to jasmine.yml file
- https://github.com/alphagov/frontend/pull/3221 - 2022
jasmine.yml file removed, app.js not moved to jamine-browser.json

Reply:
This and related code has been extracted to a separate PR.